stories/.storybook/preview.js

```diff
@@ -0,0 +1,28 @@
+import { GlobalStyle, ThemeProvider } from '@datapunt/asc-ui'
+import { addDecorator, addParameters } from '@storybook/react'
+import React from 'react'
+import { withA11y } from '@storybook/addon-a11y'
+import { DocsPage, DocsContainer } from '@storybook/addon-docs/blocks'
+import 'leaflet/dist/leaflet.css'
+import 'leaflet-draw/dist/leaflet.draw.css'
+
+function withGlobalStyles(storyFn) {
+  return (
+    <ThemeProvider>
+      <>
```

Comment:
Don't think we need this fragment, or at least `GlobalStyle` can be pulled up a level.

Reply:
Unfortunately we do need it...